dashboard/apps/prototype/static/prototype/js/main.js

```diff
@@ -191,24 +191,24 @@ var testRequest = {
   end_date: '2016-05-23'
 };
 
+function mapFloats(list, key) {
```

Comment:
i'm working on the refactoring of this, and these are reorganised in my new code. might be better to leave this change out for the time being.